cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -175,6 +180,7 @@ public Package getPackage(String orgNamePath, String packageNamePath, String ver
 
         // Set headers
         conn.setRequestProperty(BALLERINA_PLATFORM, supportedPlatform);
+        conn.setRequestProperty(USER_AGENT, ballerinaVersion);
```

Comment:
Can we move these to the `createHttpUrlConnection` function ?

Reply:
Is that the best approach? Else we can create a new method and reuse it inside all the methods. WDYT?

Reply:
I think the best way to avoid resurfacing the issues we are facing is to move them to that function. Any new function added to this class that uses the client will have User-Agent header then.

Reply:
Resolved in bf00c0696a3ebedef52553ece1642d5f705129d1